tags/cncf-tags.md

```diff
@@ -133,6 +125,54 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 * May either have no explicit roles or responsibilities, or formally assigned roles (see above).
 * May not create the impression that they have any authority or formal responsibilities in the TAG other than assigned roles.
 
+### TOC Liaisons
+
+TOC is the technical governing body of the CNCF. It’s important that we ensure a good flow of governance information and feedback loops to and from the  TAGs. With several TAGs, it can be hard to connect for community wide consensus therefore each group is assigned at least one TOC liaison.
+
+TAG leads may call on liaisons to act as a point of contact from the TOC, be an advisor for governance or community health matters, or help the TAG prioritize what they’re doing to be in line with TOC needs.
+
+Liaisons do not make decisions for the TAG.
+
+Liaisons are assigned based on their expertise, interest, and specialized experience a given TAG may need at the time of assignment (adjustments can be made, if needed). The TOC distributes TAG assignments as equal as possible. After each TOC election, liaisons are either rotated or new members will assume one of the departing members’ groups.
```

Comment:
This process is a bit shallow to me. 

> Liaisons are assigned based on their expertise, interest, and specialized experience a given TAG may need at the time of assignment (adjustments can be made, if needed).

the paragraph above contradicts the next paragraph, right?

> The TOC distributes TAG assignments as equal as possible. After each TOC election, liaisons are either rotated or new members will assume one of the departing members’ groups.

---

I think its a great idea to follow:

> Liaisons are assigned based on their expertise, interest, and specialized experience a given TAG may need at the time of assignment (adjustments can be made, if needed).

... but it is unclear how this is achived

Reply:
I think we can make this a bit more clear. TOC members may have several areas of expertise, specialized experience, or interest. So if a TOC member who a strong background in Networking and green computing was previously assigned to TAG network, they may rotate to TAG Environmental Sustainability. It could also be the case that a TAG may need a TOC Liaison with specialized experience in growing contributors and sustaining communities, but their expertise is in App Delivery, they may rotate from App Delivery to the TAG that needs to grow their own contributors or sustain their member pool and activities - not necessarily rotating to TAG Contributor Strategy.

The lack of clarity is a good call out. Today, we rely on the Chairs, Leads, activities, members, and TOC to ascertain those needs through their interactions and observations. Its not always a clear indicator, which is why establishing these roles and responsibilities is so important- so that we have better communication and understanding to make those determinations. Does this help

Reply:
```suggestion
Liaisons are assigned based on their expertise, interest, and specialized experience a given TAG may need at the time of assignment (adjustments can be made, if needed). The TOC relies on the Chairs, Leads, activities, members, and other TOC members to ascertain those needs through their interactions and observations. For instance, a TOC member with a strong background in Networking and Green Computing was previously assigned to TAG Network, they may rotate to TAG Environmental Sustainability. Alternatively, a TAG may need a TOC Liaison with specialized experience in growing contributors and sustaining communities, however their expertise is in App Delivery. In this instance they may rotate from App Delivery to the TAG that needs to grow their own contributors or sustain their member pool and activities - not necessarily rotating to TAG Contributor Strategy. 

The TOC distributes TAG assignments as equal as possible. After each TOC election, liaisons are either rotated or new members will assume one of the departing members’ groups.
```